cmd/fleet/serve.go

```diff
@@ -839,3 +847,50 @@ func getTLSConfig(profile string) *tls.Config {
 
 	return &cfg
 }
+
+// devSQLInterceptor is a sql interceptor to be used for development purposes.
+type devSQLInterceptor struct {
+	sqlmw.NullInterceptor
+
+	logger kitlog.Logger
+}
+
+func (in *devSQLInterceptor) StmtQueryContext(ctx context.Context, stmt driver.StmtQueryContext, query string, args []driver.NamedValue) (driver.Rows, error) {
+	start := time.Now()
+	rows, err := stmt.QueryContext(ctx, args)
+	in.logQuery(start, query, args, err)
+	return rows, err
+}
+
+func (in *devSQLInterceptor) StmtExecContext(ctx context.Context, stmt driver.StmtExecContext, query string, args []driver.NamedValue) (driver.Result, error) {
+	start := time.Now()
+	result, err := stmt.ExecContext(ctx, args)
+	in.logQuery(start, query, args, err)
+	return result, err
+}
+
+func (in *devSQLInterceptor) logQuery(start time.Time, query string, args []driver.NamedValue, err error) {
+	logLevel := level.Debug
+	if err != nil {
+		logLevel = level.Error
+	}
+	space := regexp.MustCompile(`\s+`)
```

Comment:
Should we move this to the outer scope as a const?

Reply:
Good idea.